src/apps/api/views/competitions.py

```diff
@@ -141,7 +141,16 @@ def update(self, request, *args, **kwargs):
         in the response to remove a GET from the frontend"""
         partial = kwargs.pop('partial', False)
         instance = self.get_object()
-        serializer = self.get_serializer(instance, data=request.data, partial=partial)
+        data = request.data
+
+        #save leaderboard indiviually, then pass pk to each phase
+        leaderboard = LeaderboardSerializer(data=data['leaderboards'][0])
```

Comment:
should we change how the frontend passes this instead?

Reply:
That's what I did in the other PR under the same name. It's much cleaner doing it this way. Unless you know of a way to pass the same leaderboard on 2 different phases without getting a duplication error.